docs/UsersGuide.asciidoc

```diff
@@ -271,6 +271,55 @@ provide a rename target, for example
 ISO_1_DECOMPRESS_URL=http://host/foo2.iso.xz and ISO_1=foo.iso
 
 
+== Asset cleanup ==
```

Comment:
Looks like you missed the introduction of the section https://github.com/os-autoinst/openQA/pull/1765/files#diff-15b5771a4f9c33cc7735eca26e0b1484R242 recently?

Reply:
It was under 'API' and I've just read the first lines. So I assumed it is only about the relevant API parameter.

Maybe it is better to have an extra section for it and have only API-specific documentation under the API section.

Reply:
sure, play around with it, move it and combine what you want to add :)